### ISSUE TITLE
Duplicate of remember-notes?

### ISSUE THREAD

**Issue by mina86** [Fanael/persistent-scratch#3]
Have you looked at remeber-notes available since 24.4? If your code has additional features I think it would be awesome to pull them to Emacs rather than having it as separate package.

**Comment by Fanael** [comment]
> Have you looked at remeber-notes available since 24.4?

Yes, and I don't see how it's similar.

> If your code has additional features I think it would be awesome to pull them to Emacs rather than having it as separate package.

Why? We have `package.el`, so there's little reason to include the kitchen sink in Emacs itself anymore.

Besides, I don't have the CLA signed and don't want to sign it, Stallman doesn't deserve my soul.

**Comment by mina86** [comment]
On Wed, Jan 21 2015, Fanael Linithien notifications@github.com wrote:

> > Have you looked at remeber-notes available since 24.4?
> 
> Yes, and I don't see how it's similar.

As far as I can see,

```
(require 'remember)
(setq remember-notes-auto-save-visited-file-name t
      remember-notes-buffer-name "*scratch*")
(setq initial-buffer-choice
      (lambda () (kill-buffer remember-notes-buffer-name)
                 (remember-notes)))
```

is almost identical to persistent-scratch-autosave-mode.  It handles
saving _scratch_ on kill as well as autosave (alas using Emacs' standard
auto-save-mode rather than custom timer).  It doesn’t save arbitrary
properties such as major-mode though, or saving _scratch_ to arbitrary
file.

> > If your code has additional features I think it would be awesome to
> > pull them to Emacs rather than having it as separate package.
> 
> Why?

Easier for users? Wider exposure? Dunno…

## 

Best regards,                                         _     _
.o. | Liege of Serenely Enlightened Majesty of      o' \,=./ `o
..o | Computer Science,  Michał “mina86” Nazarewicz    (o o)
ooo +--mpn@google.com--xmpp:mina86@jabber.org--ooO--(_)--Ooo--

**Comment by Fanael** [comment]
>  It doesn’t save arbitrary properties such as major-mode though

Yes.

> or saving _scratch_ to arbitrary file

Yes.

On top of that, it can handle only a single buffer, unlike persistent-scratch (see `persistent-scratch-scratch-buffer-p-function`); and it doesn't store backups (see `persistent-scratch-backup-directory` and `persistent-scratch-new-backup`), unless we count `make-backup-files`, which has different semantics.

Frankly, `remember-notes` looks to me like a regular buffer visiting a regular file, not much different from `(find-file "some path")`.

> Easier for users? Wider exposure?

I beg to differ, Emacs has so many built-in features already barely anybody knows about, this package would probably end up as another one. IMO, searching in `list-packages` is simply much easier than searching in NEWS files, assuming one even knows they exist.
